py/desispec/coaddition.py

```diff
@@ -77,25 +94,29 @@ def coadd_fibermap(fibermap) :
         tfmap['COADD_NUMEXP'][i] = np.count_nonzero(good_coadds)
         if 'EXPTIME' in fibermap.colnames :
             tfmap['COADD_EXPTIME'][i] = np.sum(fibermap['EXPTIME'][jj][good_coadds])
-        for k in ['DELTA_X','DELTA_Y'] :
+        for k in mean_rms_cols:
             if k in fibermap.colnames :
                 vals=fibermap[k][jj]
                 tfmap['MEAN_'+k][i] = np.mean(vals)
                 tfmap['RMS_'+k][i] = np.sqrt(np.mean(vals**2)) # inc. mean offset, not same as std
```

Comment:
This is what we want for DELTA_X DELTA_Y but probably not for the other mean_rms_cols

Reply:
Good idea.  I updated code to retain RMS for DELTA_X,Y but dropped it for the others.  MEAN is calculated for all of them, i.e.
```
    MEAN_DELTA_X        f4  
    RMS_DELTA_X         f4  
    MEAN_DELTA_Y        f4  
    RMS_DELTA_Y         f4  
    MEAN_FIBER_X        f4  
    MEAN_FIBER_Y        f4  
    MEAN_FIBER_RA       f8  
    MEAN_FIBER_DEC      f8  
    MEAN_FIBERASSIGN_X  f4  
    MEAN_FIBERASSIGN_Y  f4  
```